gcs/folder.go

```diff
@@ -158,3 +181,16 @@ func (folder *Folder) PutObject(name string, content io.Reader) error {
 	}
 	return nil
 }
+
+func (folder *Folder) joinPath(one string, another string) string {
+	if folder.normalizePrefix {
+		return storage.JoinPath(one, another)
+	}
+	if one[len(one)-1] == '/' {
+		one = one[:len(one)-1]
+	}
+	if another[0] == '/' {
+		another = another[1:]
```

Comment:
You can use strings.TrimPrefix(another, "/")

Reply:
No, I cannot, WAL-E is triming exactly one char.